pyproject.toml

```
@@ -163,41 +128,36 @@ max-line-length = 88
 [tool.pylint.messages_control]
 disable = ["C0330", "C0326"]
 
-[tool.isort]
-profile = "black"
-lines_after_imports = 2
-lines_between_sections = 1
-honor_noqa = true
-skip_gitignore = true
-skip = "pytensor/version.py"
-skip_glob = "**/*.pyx"
 
 [tool.ruff]
-select=["C","E","F","W"]
-ignore=["E501","E741","C408","C901"]
-exclude = [
-    "doc/",
-    "pytensor/_version.py",
-    "bin/pytensor_cache.py",
-]
+select = ["C", "E", "F", "I", "W"]
```

Looks like pyupgrade got dropped?

```suggestion
select = ["C", "E", "F", "I", "W", "UP"]
```

---

Good catch! Thanks! I'll resolve the issues later today 😄